vendor/json/CMakeLists.txt

```diff
@@ -1,116 +1,134 @@
 cmake_minimum_required(VERSION 3.1...3.14)
 
-##
-## PROJECT
-## name and version
-##
-project(nlohmann_json VERSION 3.11.3 LANGUAGES CXX)
-
-##
-## MAIN_PROJECT CHECK
-## determine if nlohmann_json is built as a subproject (using add_subdirectory) or if it is the main project
-##
```

Comment:
Not thrilled, but this puts the vendor cmake into the "proper" format.  Hopefully a one-time thing.